server/startup/migrations/v253.ts

```diff
@@ -0,0 +1,9 @@
+import { Permissions } from '../../../app/models/server/raw';
+import { addMigration } from '../../lib/migrations';
+
+addMigration({
+	version: 253,
+	up() {
+		return Permissions.update({ _id: 'toggle-room-e2e-encryption' }, { $addToSet: { roles: 'admin' } });
```

Comment:
```suggestion
		return upsertPermissions();
```
I think you could use the `upsertPermissions` function instead, so you don't have to update it manually (e.g. see migration 244). This should have the same effect

Reply:
Done!